jazzhands/__main__.py

```diff
@@ -196,18 +207,24 @@ def main(argv):
 
         if index_files['js'] and js_dir:
             jsx_registry_path = os.path.join(os.path.dirname(index_files['js']), 'jsx_registry.js')
-            subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
-            process_jsx(jsx_registry_path)
-            
+            try:
+                import django_jsx
+                subprocess.call(['python', 'manage.py', 'compilejsx'], stdout=open(jsx_registry_path, 'w'))
+                process_jsx(jsx_registry_path)
```

Comment:
Running 'jazzhands build' in the top dir of our travel-related site's code, it tries this command which fails since it's writing over its input:

['./node_modules/.bin/babel', '--presets=react,es2015', 'sitename/src/js/jsx_registry.js', '-o', 'sitename/src/js/jsx_registry.js']

Reply:
Ah, a little different layout than the other projects I've tested. I'll try it there and make the appropriate adjustments.